python/plotDifferentialBins.py

```diff
@@ -238,60 +247,6 @@ def plotDifferentialBins(asimovDataModel, asimovPhysicalModel, obsName, fstate,
         border_msg("Label name: {}, Unit: {}".format(label, unit))
 
 
-    # # FIXME: Also, this part we can define at one central place, probably in yaml file
-    # if (obsName=="pT4l"):
-    #     label="p_{T}^{H}"
-    #     unit="GeV"
-    # elif (obsName=="massZ2"):
-    #     label = "m(Z_{2})"
-    #     unit = "GeV"
-    # elif (obsName=="massZ1"):
-    #     label = "m(Z_{1})"
-    #     unit = "GeV"
-    # elif (obsName=="nJets" or obsName=="njets_pt30_eta4p7"):
-    #     label = "N(jets) |#eta|<4.7"
-    #     unit = ""
-    # elif (obsName=="njets_pt30_eta2p5"):
-    #     label = "N(jets) |#eta|<2.5"
-    #     unit = ""
-    # elif (obsName=="pt_leadingjet_pt30_eta4p7"):
-    #     label = "p_{T}(jet)"
-    #     unit = "GeV"
-    # elif (obsName=="pt_leadingjet_pt30_eta2p5"):
-    #     label = "p_{T}(jet) |#eta|<2.5"
-    #     unit = "GeV"
-    # elif (obsName=="absrapidity_leadingjet_pt30_eta4p7"):
-    #     label = "|y(jet)|"
-    #     unit = ""
-    # elif (obsName=="absrapidity_leadingjet_pt30_eta2p5"):
-    #     label = "|y(jet)| |#eta|<2.5"
-    #     unit = ""
-    # elif (obsName=="absdeltarapidity_hleadingjet_pt30_eta4p7"):
-    #     label = "|y(H)-y(jet)|"
-    #     unit = ""
-    # elif (obsName=="absdeltarapidity_hleadingjet_pt30_eta2p5"):
-    #     label = "|y(H)-y(jet)| |#eta|<2.5"
-    #     unit = ""
-    # elif (obsName=="rapidity4l"):
-    #     label = "|y^{H}|"
-    #     unit = ""
-    # elif (obsName=="cosThetaStar"):
-    #     label = "cos#theta*"
-    #     unit = ""
-    # elif (obsName=="cosTheta1"):
-    #     label = "cos#theta_{1}"
-    #     unit = ""
-    # elif (obsName=="cosTheta2"):
-    #     label = "cos#theta_{2}"
-    #     unit = ""
-    # elif (obsName=="Phi"):
-    #     label = "#Phi"
-    #     unit = ""
-    # elif (obsName=="Phi1"):
-    #     label = "#Phi_{1}"
-    #     unit = ""
-
-
     if (obsName.startswith("njets")):
```

Comment:
Shouldn't this be also for 2D case? #24

Reply:
Will add this into another issue as you mentioned that 2D part is still not implemented here.